wasmcloud_host/lib/wasmcloud_host/lattice/state_monitor.ex

```diff
@@ -61,7 +61,7 @@ defmodule WasmcloudHost.Lattice.StateMonitor do
 
     state =
       cond do
-        String.ends_with?(topic, ".events") ->
+        String.contains?(topic, ".evt") ->
```

Comment:
Should this be `.evt.` to ensure it's in the middle? Or perhaps split by period and compare the second value to `evt`?

Reply:
Just wouldn't want someone's lattice prefix to be `evt` and trigger this.

Reply:
That looks good, I get wary with `contains` since that can pretty easily match on unintended strings. Change LGTM